Source/Core/Common/Logging/ConsoleListener.h

```diff
@@ -12,7 +12,4 @@ class ConsoleListener : public Common::Log::LogListener
   ~ConsoleListener();
 
   void Log(Common::Log::LogLevel level, const char* text) override;
-
-private:
-  bool m_use_color = false;
```

Comment:
I would just mark this as `[[maybe_unused]]`, so we don't introduce more mutable global state and make it easier for other platforms to make use of it at the same time.

Reply:
Done.